projects/super_res/fv3net.yml

```diff
@@ -0,0 +1,297 @@
+name: fv3net
```

Comment:
I see that this has more specified deps, which makes it more brittle for others but makes your workflow a bit faster. I hope that you won't need to do this step very often anyway but in the meantime -- could you help me understand why this is called fv3net? Is just the deps for your code?

Reply:
yeah yeah I just named my env that way, no specific reason

Reply:
nit: I think it's a bit confusing. Best change it to something like `vsr`.